test/BooleanVar.py

```diff
@@ -21,6 +21,32 @@ def testBindFalse(self):
                 (False,))
         self.assertEqual(result, "FALSE")
 
+    def testBindFloatAsBoolean(self):
+        "test binding in a float as a boolean"
+        var = self.cursor.var(cx_Oracle.BOOLEAN)
```

Comment:
Please use `var = self.cursor.var(bool)` instead. `cx_Oracle.BOOLEAN` is deprecated.

Reply:
If you prefer, I can make the changes after I merge as well.

Reply:
Done.